files/.github/workflows/ci.yml

```diff
@@ -42,6 +42,9 @@ jobs:
           cache: <%= yarn ? 'yarn' : 'npm' %>
       - name: Install Dependencies
         run: <%= yarn ? 'yarn install --no-lockfile' : 'npm install --no-shrinkwrap' %>
+      - name: Build Addon
+        run: <%= yarn ? 'yarn' : 'npm run' %> build
+        working-directory: <%= name %>
```

Comment:
Shouldn't this be like this?

```suggestion
        working-directory: packages/<%= addonName %>
```

Reply:
this is news to me. I haven't been using this blueprint (because it doesn't fit how I've been making v2 addons), but I imagine we'll want the whole path to be configurable.

Reply:
updated

Reply:
`packages/` is still missing here!? And `<%= name %>` produces `dummy`! 😆

So my suggestion above should be what is needed here (and in L80).

You should try running the blueprint off your branch (`ember addon my-addon -b NullVoxPopuli/addon-blueprint --yarn --skip-npm`) to make sure it produces the expected output!

Reply:
more PRs to come, I promise.

Reply:
> why is <%= name %> dummy? that shouldn't exist :-\

Seems there was still some c&p leftovers from the original blueprint here: https://github.com/embroider-build/addon-blueprint/pull/17/files/3809c8359822a6ad3b4130435c9318118e845014..4ee5bf85d8e5e267f345311506cfd25988e553b4#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R95. Needs to go away...